src/diffusion_for_multi_scale_molecular_dynamics/models/score_networks/score_network.py

```diff
@@ -1,8 +1,12 @@
-"""Score Network.
+r"""Score Network.
 
 This module implements score networks for positions in relative coordinates.
 Relative coordinates are with respect to lattice vectors which define the
 periodic unit cell.
+
+The coordinates part of the output aims to calculate
+    output.X \propto nabla_X \ln P(x,t)
```

Comment:
if you use an "r" string, you should use the equation markup
... math::
Otherwise, remove the "r" string

Reply:
done.